### PR TITLE
Specified additional item requirements in two clue scrolls

### LINKED ISSUE #11158: Missing Clue Helper Items
There are some clue helpers that don't list all of the required/reccomended items. Two that I know of are:

- the elite coordinate clue for north-east of dragontooth island does not state you need a ghostspeak amulet AND 25 ecto tokens to get there. It would be helpful to have this information beforehand.

- the master cryptic clue for Viyeldi caves doesn't mention that you may need an axe and machete. It has all of the other items like the lockpick but it is very annoying when you get there with no axe or machete to get through the dense forest. It should say you need them if you don't have the agility level to hop over.

There may be others like this but I am unaware of them at the moment.

### PR DESCRIPTION
**Pull request by TheFlemoid**
FIxes #11158 

Specified additional item requirements in Dragontooth Island elite coordinate clue
Specified additional item requirements in Viyeldi Caves master cryptic clue

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java [hunk 1]
```diff
@@ -230,7 +230,7 @@
 		new CrypticClue("Dig under Razorlor's toad batta.", new WorldPoint(3139, 4554, 0), "Dig on the toad batta spawn in Tarn's Lair."),
 		new CrypticClue("Talk to Cassie in Falador.", "Cassie", new WorldPoint(2975, 3383, 0), "Cassie is found just south-east of the northern Falador gate."),
 		new CrypticClue("Faint sounds of 'Arr', fire giants found deep, the eastern tip of a lake, are the rewards you could reap.", new WorldPoint(3055, 10338, 0), "Dig south of the pillar in the Deep Wilderness Dungeon in the room with the fire giants."),
-		new CrypticClue("If you're feeling brave, dig beneath the dragon's eye.", new WorldPoint(2410, 4714, 0), "Dig below the mossy rock under the Viyeldi caves (Legend's Quest). Items needed: Pickaxe, unpowered orb, lockpick, spade, and any charge orb spell."),
+		new CrypticClue("If you're feeling brave, dig beneath the dragon's eye.", new WorldPoint(2410, 4714, 0), "Dig below the mossy rock under the Viyeldi caves (Legend's Quest). Items needed: Pickaxe, unpowered orb, lockpick, spade, and any charge orb spell. If under 79 agility you will need an axe and machete."),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java:233]:
```suggestion
		new CrypticClue("If you're feeling brave, dig beneath the dragon's eye.", new WorldPoint(2410, 4714, 0), "Dig below the mossy rock under the Viyeldi caves (Legend's Quest). Items needed: Pickaxe, unpowered orb, lockpick, spade, any charge orb spell, and either 79 agility or an axe and machete."),
```

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java [hunk 2]
```diff
@@ -175,7 +175,7 @@ private CoordinateClueInfo(@Nonnull String directions, boolean lightRequired, Va
 		.put(new WorldPoint(3828, 2848, 0), new CoordinateClueInfo("East of Harmony Island."))
 		.put(new WorldPoint(3225, 2838, 0), new CoordinateClueInfo("South of Desert Treasure pyramid."))
 		.put(new WorldPoint(1773, 3510, 0), new CoordinateClueInfo("Ruins north of the Hosidius mine."))
-		.put(new WorldPoint(3822, 3562, 0), new CoordinateClueInfo("North-east of Dragontooth Island."))
+		.put(new WorldPoint(3822, 3562, 0), new CoordinateClueInfo("North-east of Dragontooth Island. Bring Ghostspeak Amulet and 25 ecto tokens to get to the island."))
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java:178]:
```suggestion
		.put(new WorldPoint(3822, 3562, 0), new CoordinateClueInfo("North-east of Dragontooth Island. Bring a Ghostspeak Amulet and 25 Ecto-tokens to get to the island."))
```

**Comment by makingcomment** [comment]
It only costs 25 Ecto-tokens to reach Dragontooth Island if you haven't paid the larger fee for infinite trips.

**Comment by Nightfirecat** [comment]
That's true, but players who have done that would (hopefully) remember that and know not to bring any. Or if they didn't, they'd be fine as well.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java (hunk 2 above)
Comment (TheFlemoid) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java]: Corrected

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java (hunk 1 above)
Comment (TheFlemoid) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java]: Corrected

**Comment by TheFlemoid** [comment]
Sorry for the delay, works gotten busy in the midst of this pandemic thing.  Rebased the PR due to new version, and made changes requested by @Nightfirecat.

**Pull request merged**